src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -236,7 +236,12 @@
         /*
         Whether Harvesting (OAI) service is enabled
         */
-        OAIServerEnabled;
+        OAIServerEnabled,
+        
+        /**
+        * Whether Shibboleth passive authentication mode is enabled
+        */
+       ShibPassiveLoginEnabled;
```

Comment:
Ideally, you'd document this new database setting so that it appears (near the bottom) of a future version of http://guides.dataverse.org/en/4.6.1/installation/config.html#database-settings